hacksquad_discordpy/main.py

```diff
@@ -20,9 +26,12 @@ async def help(ctx):
     await ctx.send("Help")
 
 @bot.command()
+@commands.has_permissions(administrator=True)
 async def sync(ctx):
+    await ctx.send("Syncing the bot...")
     await bot.tree.sync()
     await ctx.send("Synced")
```

Comment:
It would be nice to edit the message after it is synced rather than sending a new one